environment.yml

```diff
@@ -0,0 +1,18 @@
+name: brdnet
+channels:
+    - conda-forge
+    - r
+    - bioconda
+dependencies:
+    - r::r=3.5.1
```

Comment:
Is this syntax (channel::package=version) necessary here, or was it just the default when you ran `conda env export`?

If it's necessary, you might want to specify in your README that it doesn't always work with older versions of conda (see, for example, https://github.com/conda/conda/issues/2800). If it was just the default, don't worry about it, what you have is fine.

Reply:
I added it to be more explicit since I had dependencies coming from multiple different channels. I didn't know that was a newer addition to conda though, I'll definitely add a note in the README.